### ISSUE TITLE
web gui not working on current main

### ISSUE THREAD

**Issue by ik666** [UniversalMediaServer/UniversalMediaServer#4757]
@SurfaceS It looks like the current main doesn't bundle the UI in the distribution package. I've got a blanc screen. There are actually 2 things:

1) after calling `mvn package` the ums GZIP distribution package doesn't contain the frontend. Instead, it's located in the `web/react-client` folder (if you untar the GZ file). I moved the files into the ums.jar file and now ...

2) The generated javascript and css files are being referred case sensitive. The index.html includes for example `/static/index-DyoNl0AC.css`, however the `WebGuiServlet` rewrites the URI to lowercase. Therefore, `writeAsync` fails delivering the resource.

```
			String uri = req.getRequestURI() != null ? req.getRequestURI().toLowerCase() : "/index.html";
			if (uri.equals(BASE_PATH) || ROUTES.contains(uri) || uri.startsWith(PLAYER_BASE_PATH)) {
				uri = "/index.html";
			}
			if (uri.startsWith("/static/")) {
				resp.setHeader("Cache-Control", "public, max-age=604800");
			}
			if (!writeAsync(req, resp, uri.substring(1))) {
				// The resource manager can't found or send the file, we need to send a response.
				LOGGER.trace("WebGuiServlet request not available : {}", req.getRequestURI());
				respond(req, resp, "<html><body>404 - File Not Found: " + req.getRequestURI() + "</body></html>", 404, "text/html");
			}

```

**Comment by SurfaceS** [comment]
`web/react-client` folder is the normal base for react client.
it should sit under `umsfolder/web/react-client` after installation.

this `req.getRequestURI().toLowerCase()` may be a problem on linux/mac systems.

**Comment by SurfaceS** [comment]
fixed in #4758

**Comment by ik666** [comment]
OK, thx. I merged the fix, but still needed to add the full path `-Xbootclasspath/a:/UniversalMediaServer/web/react-client` as VM options for the UI to load.
